src/Tools/dotnet-trace/CommandLine/Commands/CollectLinuxCommand.cs

```diff
@@ -0,0 +1,325 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Diagnostics.NETCore.Client;
+using Microsoft.Diagnostics.Tools.Common;
+using Microsoft.Internal.Common.Utils;
+
+namespace Microsoft.Diagnostics.Tools.Trace
+{
+    internal partial class CollectLinuxCommandHandler
+    {
+        private bool stopTracing;
+        private Stopwatch stopwatch = new();
+        private LineRewriter rewriter;
+        private bool printingStatus;
+
+        internal sealed record CollectLinuxArgs(
+            CancellationToken Ct,
+            string[] Providers,
+            string ClrEventLevel,
+            string ClrEvents,
+            string[] PerfEvents,
+            string[] Profiles,
+            FileInfo Output,
+            TimeSpan Duration);
+
+        public CollectLinuxCommandHandler(IConsole console = null)
+        {
+            Console = console ?? new DefaultConsole(false);
+            rewriter = new LineRewriter(Console);
+        }
+
+        /// <summary>
+        /// Collects diagnostic traces using perf_events, a Linux OS technology. collect-linux requires admin privileges to capture kernel- and user-mode events, and by default, captures events from all processes.
+        /// This Linux-only command includes the same .NET events as dotnet-trace collect, and it uses the kernel’s user_events mechanism to emit .NET events as perf events, enabling unification of user-space .NET events with kernel-space system events.
+        /// </summary>
+        internal int CollectLinux(CollectLinuxArgs args)
+        {
+            if (!OperatingSystem.IsLinux())
+            {
+                Console.Error.WriteLine("The collect-linux command is only supported on Linux.");
+                return (int)ReturnCode.PlatformNotSupportedError;
+            }
+
+            Console.WriteLine("==========================================================================================");
+            Console.WriteLine("The collect-linux verb is in preview. Some usage scenarios may not yet be supported,");
```

Comment:
I doubt the average user will know what "NetTrace V6" means :) They probably recognize the file suffix but not the "V6". I'd also suggest we can make the message a little useful by telling them what will work and pointing them at the docs. To keep the message from getting too long we can put bug filing info as part of a longer message we include in the docs. How about this:

"The collect-linux verb is a new preview feature and relies on an updated version of the .nettrace file format. The latest PerfView release supports these trace files but other ways of using the trace file may not work yet. See the docs at https://learn.microsoft.com/dotnet/core/diagnostics/dotnet-trace for more details."

Reply:
Added, I've been separating the lines to where each line is just under 90 characters, which is the width of
 `Provider Name                           Keywords            Level               Enabled By`
 so slightly adjusted